.config/eslint.mjs

```diff
@@ -0,0 +1,581 @@
+// -----------------------------------------------------------------------------
+// Requirements
+// -----------------------------------------------------------------------------
+
+const configEntries = ['.eslintrc.js'];
+const prettierOptions = {
+  bracketSpacing: true,
+  endOfLine: 'lf',
+  printWidth: 80,
+  tabWidth: 2,
+  useTabs: false,
+  singleQuote: true,
+  overrides: [
+    {
+      files: [
+        '*.cjs',
+        '*.js',
+        '*.mjs',
+      ],
+      options: {
+        arrowParens: 'always',
+        bracketSpacing: true,
+        parser: 'typescript',
+        printWidth: 80,
+        quoteProps: 'as-needed',
+        semi: true,
+        singleQuote: true,
+        tabWidth: 2,
+        trailingComma: 'es5',
+        useTabs: false,
+      },
+    },
+    {
+      files: '*.md',
+      options: {
+        proseWrap: 'always',
+        printWidth: 80,
+        parser: 'markdown',
+      },
+    },
+  ],
+};
+
+const OFF = 0;
+const WARN = 1;
+const ON = 2;
+
+const definedTypes = [
+  'IMapLike',
+  'SharedArrayBuffer',
+  'CjsModule',
+  'Es6Module',
+];
+
+// -----------------------------------------------------------------------------
+// Data
+// -----------------------------------------------------------------------------
+
+export default {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: [
+    // Order matters; don't alphasort. Rules of later presets override previous.
+    'eslint:recommended',
+    'semistandard',
```

Comment:
> [&hellip;] the practices that they encourage, such as leaving out semicolons, means that parsers do double work when parsing ambiguous statements.
> &mdash; https://github.com/tc39/tc39.github.io/pull/213/files#r505296951

@codehag, I did not know this, but I learned Java first and prefer the semicolons for clarity.

Omitting semicolons is also a bad idea for many other reasons too.

In the comment above I stated the following&hellip;

> This config in this PR is actually using standard (actually semi-standard).

When I said "semi-standard", I messed up and should've said "semistandard".

Sorry, still not too familiar w/ it, but here's what that is:

> All the goodness of standard/standard with semicolons sprinkled on top.
> &mdash; https://github.com/standard/semistandard